code/datums/objective/assasinate_marshal.dm

```diff
@@ -1,30 +1,35 @@
 /datum/objective/assassinate/marshal
 
+
 /datum/objective/assassinate/marshal/get_panel_entry()
 	var/target = src.target ? "[src.target.current.real_name], the [src.target.assigned_role]" : "no_target"
-	return "Find and assassinate <a href='?src=\ref[src];switch_target=1'>[target]</a>."
+	return "You are after the fugitive <a href='?src=\ref[src];switch_target=1'>[target]</a>."
 
 /datum/objective/assassinate/marshal/get_info()
 	if(target)
 		return "(target was [target.name])"
 
 /datum/objective/assassinate/marshal/update_explanation()
 	var/target_role = ""
-	if(target.antagonist.len)
+	if(target && target.antagonist.len)
 		for(var/datum/antagonist/A in target.antagonist)
 			if(!A.outer)
 				target_role = A.role_text
 				break
 
 	if(target && target.current)
-		explanation_text = "Find and assassinate the [target_role]."
+		explanation_text = "Find and take down the [target_role]."
 	else
 		explanation_text = "Target has not arrived today. Is it a coincidence?"
 
-/datum/objective/assasinate/marshal/get_targets_list()
+/datum/objective/assassinate/marshal/get_targets_list()
 	var/list/possible_targets = list()
 	for(var/datum/mind/possible_target in SSticker.minds)
-		if(possible_target != owner && ishuman(possible_target.current) && (possible_target.current.stat != 2) && \
- 		player_is_ship_antag(possible_target))
+		if(possible_target != owner &&\
+		 ishuman(possible_target.current) &&\
+		  (possible_target.current.stat != 2) &&\
+		   (player_is_ship_antag(possible_target)))
```

Comment:
Is there something wrong with indents here?

Reply:
these are continued lines, note the \
thats really all one line of code